src/core/public/chrome/ui/header/recent_items.tsx

```diff
@@ -94,46 +182,98 @@ export const RecentItems = ({
     </EuiToolTip>
   );
 
+  useEffect(() => {
+    const savedObjects = recentlyAccessedItems
+      .filter((item) => item.meta?.type)
+      .map((item) => ({
+        type: item.meta?.type || '',
+        id: item.id,
+      }));
+
+    if (savedObjects.length) {
+      bulkGetDetail(savedObjects, http).then((res) => {
```

Comment:
Why should it call this API to get the details? Isn't items from `recentlyAccessed$` already have saved object title, type and workspace?

Reply:
Hi, Thanks for your review, we call the API to get the associated icon